list

```diff
@@ -326,6 +326,7 @@ flic.kr
 flip.it
 flomuz.io
 flq.us
+flx.to
```

Comment:
Thanks for your contribution @ignoramous 🎉

Looks like its currently not resolving, also I didn't find a real use case of it, would you like to provide more information about this domain?

Reply:
sure, `flx.to/flexport-app1`: https://archive.is/bvw27